packages/react-native-editor/ios/Podfile.lock

```diff
@@ -542,10 +542,10 @@ SPEC CHECKSUMS:
   RNReanimated: d87c75f1076bab3402d6cd0b7322be51d333d10e
   RNScreens: 953633729a42e23ad0c93574d676b361e3335e8b
   RNSVG: 36a7359c428dcb7c6bce1cc546fbfebe069809b0
-  RNTAztecView: 0783a43eb5241e38587a52cb4eadc0ef2852607a
+  RNTAztecView: def74944705c4bef636fc653d11eaa1f49b64c22
   WordPress-Aztec-iOS: 7d11d598f14c82c727c08b56bd35fbeb7dafb504
   Yoga: 9a08effa851c1d8cc1647691895540bc168ea65f
 
 PODFILE CHECKSUM: d0a2d4714ee19a1821eb2a30029333be8d6a729f
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.11.3
```

Comment:
Looks like the Cocoapods version has changed but the `Gemfile.lock` points to the previous version:
https://github.com/WordPress/gutenberg/blob/78cea0f53cc25266e9dba08e49a3af8d93cfe8bc/packages/react-native-editor/ios/Gemfile.lock#L17

I'm wondering whether we should update it 🤔 . In any case, if we do it, we should also update the `Gemfile` file too:
https://github.com/WordPress/gutenberg/blob/78cea0f53cc25266e9dba08e49a3af8d93cfe8bc/packages/react-native-editor/ios/Gemfile#L3

Reply:
Nice catch, I reverted it